admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -161,6 +161,23 @@ In case of UTF8MB4 you will also find::
 
     "mysql.utf8mb4" => true,
 
+SSL for MySQL Database
+^^^^^^^^^^^^^^^^^^^^^^
+
+This just covers the SSL database configuration on the Nextcloud server. First you need to configure your database server accordingly.
+
+::
+
+  'dbdriveroptions' => [
+    1007 => '/../ssl-key.pem',
+    1008 => '/../ssl-cert.pem',
+    1009 => '/../ca-cert.pem',
+    1014 => true,
```

Comment:
Also those are all are not the correct ones.

![Bildschirmfoto 2020-10-28 um 13 26 55](https://user-images.githubusercontent.com/245432/97435827-48260b00-1921-11eb-8461-024aa82145f4.png)

Reply:
Oh dear. Then I am baffeld why it works though....